test/Tester/OrleansLoggingTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Orleans.Extensions.Logging;
+using Orleans.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Tester
+{
+    [TestCategory("BVT"), TestCategory("OrleansLogging")]
+    public class OrleansLoggingTests
+    {
+        [Fact]
+        [Obsolete]
+        public void OrleansLoggingCanConfigurePerCategoryServeriyOverrides()
+        {
+            //configure logging with severity overrides
+            IServiceCollection serviceCollection = new ServiceCollection();
+            serviceCollection.AddLogging();
+            var severityOverrides = new OrleansLoggerSeverityOverrides();
+            severityOverrides.LoggerSeverityOverrides.TryAdd(this.GetType().FullName, Severity.Warning);
+            serviceCollection.AddLogging(builder => builder.AddOrleansLogging(new List<ILogConsumer>()
+            {
+                new FileLogConsumer($"{this.GetType().Name}.log")
+            }, severityOverrides));
+            var serviceProvider = serviceCollection.BuildServiceProvider();
+            //get logger
+            var logger = serviceProvider.GetRequiredService<ILogger<OrleansLoggingTests>>();
+            Assert.True(logger.IsEnabled(LogLevel.Warning));
+            Assert.False(logger.IsEnabled(LogLevel.Information));
+
+            //dispose log providers
+            (serviceProvider as IDisposable)?.Dispose();
```

Comment:
instead of this line, wrap the `serviceProvider` into a `using()` statement. Same with other facts.